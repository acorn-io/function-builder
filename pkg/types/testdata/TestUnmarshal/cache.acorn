targets: all: {
	run: [
		{
			command: "hi"
			mount: cache: "/foo"
		},
		{
			command: "hi"
			mount: [
				{
					cache: {
						target: "/foo"
					}
				},
			]
		},
		{
			command: "hi"
			mount: [
				{
					cache: "/foo"
				},
			]
		},
	]
}
