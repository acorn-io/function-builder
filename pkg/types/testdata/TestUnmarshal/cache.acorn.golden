{
  "targets": {
    "all": {
      "from": "scratch",
      "workdir": "/src",
      "run": [
        {
          "command": "hi",
          "mount": [
            {
              "cache": {
                "target": "/foo"
              }
            }
          ]
        },
        {
          "command": "hi",
          "mount": [
            {
              "cache": {
                "target": "/foo"
              }
            }
          ]
        },
        {
          "command": "hi",
          "mount": [
            {
              "cache": {
                "target": "/foo"
              }
            }
          ]
        }
      ]
    }
  }
}